tests/components/konnected/test_init.py

```diff
@@ -250,6 +250,74 @@ def mock_constructor(host, port, websession):
     assert len(hass.config_entries.flow.async_progress()) == 1
 
 
+async def test_setup_multiple(hass):
+    """Test we initiate config entry for multiple panels."""
+    assert (
+        await async_setup_component(
+            hass,
+            konnected.DOMAIN,
+            {
+                konnected.DOMAIN: {
+                    konnected.CONF_ACCESS_TOKEN: "arandomstringvalue",
+                    konnected.CONF_API_HOST: "http://192.168.86.32:8123",
+                    konnected.CONF_DEVICES: [
+                        {
+                            konnected.CONF_ID: "aabbccddeeff",
+                            "binary_sensors": [
+                                {
+                                    "zone": 4,
+                                    "type": "motion",
+                                    "name": "Hallway Motion",
+                                },
+                                {
+                                    "zone": 5,
+                                    "type": "window",
+                                    "name": "Master Bedroom Window",
+                                },
+                                {
+                                    "zone": 6,
+                                    "type": "window",
+                                    "name": "Downstairs Windows",
+                                },
+                            ],
+                            "switches": [{"zone": "out", "name": "siren"}],
+                        },
+                        {
+                            konnected.CONF_ID: "445566778899",
+                            "binary_sensors": [
+                                {"zone": 1, "type": "motion", "name": "Front"},
+                                {"zone": 2, "type": "window", "name": "Back"},
+                            ],
+                            "switches": [
+                                {
+                                    "zone": "out",
+                                    "name": "Buzzer",
+                                    "momentary": 65,
+                                    "pause": 55,
+                                    "repeat": 4,
+                                },
+                            ],
+                        },
+                    ],
+                }
+            },
+        )
+        is True
+    )
+
+    # Flow started for discovered bridge
+    assert len(hass.config_entries.flow.async_progress()) == 2
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5054188)